scripts/techsoup/us_usda/Summer_Meal_Sites_2022_v0.tmcf

```diff
@@ -0,0 +1,22 @@
+Node: E:Data->E1
+typeOf: UsdaSummerMealSite
+day: E:Data->E2
+longitude: C:Data->X
+latitude: C:Data->Y
+city: C:Data->siteCity
+state: C:Data->siteState
+zip: C:Data->siteZip
+county: C:Data->County
+season: C:Data->Season
+
+Node: E:Data->E2
+dcid: <summer-meal-site-dcid>-<day>
```

Comment:
The CSV should have two more columns: 1) for this DCID; 2) for the meal site dcid